github_issue_summarization/02_training_the_model_tfjob.md

```diff
@@ -38,24 +41,24 @@ Submit the training job
 ks apply --env=${KF_ENV} -c tfjob-pvc
 ```
 
-The resulting model will be stored on PVC so to access it you will
-need to run a pod and attach the PVC. For serving you can just
-attach it the pod serving the model.
+The resulting model will be stored on the PVC, so to access it you will
+need to run a pod and attach the PVC. For serving, you can just
+attach it to the pod serving the model.
 
 ## Training Using GCS
 
-If you are running on GCS you can train using GCS to store the input
+If you are using GCS, you can train using GCS to store the input
 and the resulting model.
 
-### GCS Service account
+### GCS service account
 
-* Create a service account which will be used to read and write data from the GCS Bucket.
+* Create a service account that will be used to read and write data from the GCS bucket.
```

Comment:
If they used kfctl.sh or click to deploy there should already be a service account in the namespace "kubeflow" that they could use.

However if they wanted to use a different namespace they would need to perform these steps.